samples/ingestion/ingestion-client/FetchTranscription/FetchTranscriptionEnvironmentVariables.cs

```diff
@@ -59,5 +66,26 @@ public static class FetchTranscriptionEnvironmentVariables
         public static readonly bool CreateAudioProcessedContainer = bool.TryParse(Environment.GetEnvironmentVariable(nameof(CreateAudioProcessedContainer), EnvironmentVariableTarget.Process), out CreateAudioProcessedContainer) && CreateAudioProcessedContainer;
 
         public static readonly string AudioProcessedContainer = Environment.GetEnvironmentVariable(nameof(AudioProcessedContainer), EnvironmentVariableTarget.Process);
+
+        public static List<ConversationPiiCategory> GetConversationPiiCategories(string piiCategories)
+        {
+            if (string.IsNullOrWhiteSpace(piiCategories))
+            {
+                return new List<ConversationPiiCategory>
+                {
+                    ConversationPiiCategory.All
+                };
+            }
+
+            var piiCategoriesList = piiCategories.Split(',').ToList();
+            var categories = new List<ConversationPiiCategory>();
+            piiCategoriesList.ForEach(category =>
+            {
+                Enum.TryParse(category, ignoreCase: true, out ConversationPiiCategory catgoryEnum);
```

Comment:
if TryParse would return false I believe categoryEnum would be null, so we shouldn't add it

Reply:
Removed the enum as discussed in a separate thread, we do not want to declare categories as enum as that will require clients to re-deploy the ARM template when we add more PII types.